internal-services/catalog/create-advisory-task.yaml

```diff
@@ -112,9 +115,14 @@ spec:
           advisoryJsonWithKey=$(jq -c --arg key "$signingKey" \
             '.content.images[] += {"signingKey": $key}' <<< '$(params.advisory_json)')
 
+          # Prepare variables for the advisory template
+          DATA=$(jq -c '{"advisory":{"spec":.}' <<< $advisoryJsonWithKey)
+          DATA=$(jq -c --arg advisory_name "$ADVISORY_NAME" --arg advisory_ship_date "$SHIP_DATE" \
```

Comment:
If this actually works, it looks cool

Reply:
Yeah, I started adding it to the command below and it was getting very long, so I came up with this. The first one just wraps the `$advisoryJsonWithKey` json with advisory.spec and the second adds the two other properties.

Reply:
Sweet, I've never seen the `$ARGS.named` syntax

Reply:
Me neither. I saw it randomly when googling how to compose a json from scratch using jq.

Reply:
I will be testing the advisory stuff end to end soon enough anyways so we will see how it goes :slightly_smiling_face: